frontend/src/components/CANs/CANFundingYTD/CANFundingYTD.jsx

```diff
@@ -26,6 +33,14 @@ const CANFundingYTD = ({ total_funding = "0", current_funding = "0", expected_fu
     ];
     const [activeId, setActiveId] = React.useState(0);
 
+    const CarryForwardTag = ({ funding, label }) => {
+        if (funding > 0) {
```

Comment:
I forget, did we have a use-case where the carry forward could legit be negative (indicating an over spending)? I might be thinking of something else.

Reply:
This is just rendering the tag for those cans that have carry-forward ( greater-than 0), vs those that don't (equal-to 0).